.github/workflows/weekly_check.yml

```diff
@@ -19,17 +19,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: ['ubuntu-20.04', 'windows-2022']
-        python-version: ['3.10']
+        python-version: ['3.9', '3.10', '3.11']
+        extra: ['default', 'tf', 'tfds', 'torch']
         include:
+          - python-version: "3.9"
+            tox-env-py: "39"
           - python-version: "3.10"
             tox-env-py: "310"
-          - os: "ubuntu-20.04"
-            tox-env-os: "lin"
-          - os: "windows-2022"
-            tox-env-os: "win"
-    name: stability test (${{ matrix.os }}, Python ${{ matrix.python-version }})
-    runs-on: ${{ matrix.os }}
+          - python-version: "3.11"
+            tox-env-py: "311"
+    name: stability test (Python ${{ matrix.python-version }}, Extra ${{ matrix.extra }})
+    runs-on: ubuntu-latest
```

Comment:
can I ask you why testing on different `os` is dropped?

Reply:
we're using fuzzing tool `atheris` but they does not support windows.

Reply:
Is this okay to skip tests for windows?

Reply:
current fuzzing implementation is very primitive and works only with datumaro CLI. but I can say that this changes would cover small portion of stability tests that required to this product. it's better to work as well for the windows but it's not supported by a tool of fuzzing we're using.